ocaml/xapi/repository.ml

```diff
@@ -51,13 +51,16 @@ let introduce ~__context ~name_label ~name_description ~binary_url ~source_url
 let introduce_bundle ~__context ~name_label ~name_description =
```

Comment:
This was probably introduced in a different PR, but I've downloaded the generated SDK from the gihtub actions of the current PR and the release info for `repository.introduce_bundle` is `1.301.0`, which looks like a copy-paste error from `repository.introduce`.

Reply:
This is a build issue. The reason is below:
The release info is based on a field `lifecycle` in the code. As the code is still in review and being tested, we don't know what value should it be, so the `lifecycle` is kept empty.
After the code is merged into the feature branch and the testing is finished, we will then merge the code into the master branch. At this time point, we will change the `lifecycle` to the actual release info such as `24.23.0`.
For the koji build, it will execute `make build` first (if the `lifecycle` is empty, it will auto-fill a temporary value such as `24.21.0-next` into `lifecycle`), and then execute `make sdk` to generate the SDK.
But for the PR build, `make build` and `make sdk` are executed concurrently, so the `lifecycle` is still empty when the SDK is generated, this leads the release info to be an old previous version number.

The koji-built SDK doesn't have this issue, you can download from here which is built based on this PR this morning.
https://koji.eng.citrite.net/kojifiles/repos/pb-t_bengangy-non-cdn/231372/x86_64/toplink/packages/xapi/24.21.0.17.gc870b266b/1.2.g8796faa.xs8/noarch/xapi-sdk-24.21.0.17.gc870b266b-1.2.g8796faa.xs8.noarch.rpm